Sources/TuistGenerator/Linter/TargetActionLinter.swift

```diff
@@ -23,8 +23,10 @@ class TargetActionLinter: TargetActionLinting {
     ///
     /// - Parameter action: Action to be linted.
     /// - Returns: Found linting issues.
```

Comment:
Maybe I'd add lint whether the `embeddedScript` is empty or not -> having an empty script could be unexpected

Reply:
I haven't explored the effects of the linter yet, but will lint failures yield a failed project generation? If so I'm not sure this is what I'd want to see as a consumer. If I put in an empty script then an empty phase would be what I'd expect to see in my project. If we have a way of warning users that this is an empty step then I'm good with that. I just don't want to fail creating the project because of an empty script.

Reply:
I agree, `LintingIssue` has a parameter `severity` to achieve that 🙂

Reply:
This is done.